eng/Version.Details.xml

```diff
@@ -8,5 +8,20 @@
       <Sha>80b6be47e1425ea90c5febffac119250043a0c92</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.22616.1">
```

Comment:
We are inconsistent across the repos in regards to this being a tooling versus product dependency.  @mmitche, we've talked about this in the past.  Now that you know more about source-build what should this be classified as?  I think it should be a product dependency.

Reply:
I think it should be product too. It may not make a ton of difference, but product dependencies have more restrictions (can't have cycles, show up in coherency calculations in BARViz).

Reply:
Why is this package name using `-` for separators?

Reply:
Maybe @MichaelSimons could help answer that

Reply:
> Why is this package name using `-` for separators?

Because the naming pattern for the intermediate packages is Microsoft.SourceBuild.Intermediate.<repo name>.  The repo name is literally [source-build-reference-packages](https://github.com/dotnet/source-build-reference-packages).  This is a internal non-shipping infrastructure package.

Reply:
Friendly reminder @oleksandr-didyk to move this to be a product dependency.

Reply:
> > Why is this package name using `-` for separators?
> 
> Because the naming pattern for the intermediate packages is Microsoft.SourceBuild.Intermediate.. The repo name is literally [source-build-reference-packages](https://github.com/dotnet/source-build-reference-packages). This is a internal non-shipping infrastructure package.

Generally, we don't name packages based on name of repos. Not that this is super important, but the package could have been named `Microsoft.SourceBuild.Intermediate.SourceBuildReferences` or somethign like that.

Reply:
> Friendly reminder @oleksandr-didyk to move this to be a product dependency.

My bad, committed the change but did not push. Thanks